dev/drivers/scripts/cam/plots/jevs_cam_nam_firewxnest_grid2obs_plots.sh

```diff
@@ -10,7 +10,7 @@
 
 export model=evs
 
-export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS_firewx_fixes/EVS
```

Comment:
And you konw the drill here ... revert to ${USER} for HOMEevs.

Reply:
Done.